packages/appChrome/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "0.0.0-dev+semantically-released",
+  "description": "",
+  "keywords": [
+    "react",
+    "components"
+  ],
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index",
+  "homepage": "https://github.com/dcos-labs/ui-kit#readme",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dcos-labs/ui-kit.git"
+  },
+  "dependencies": {
+    "@dcos/ui-kit-clickable": "^0.0.0-dev+semantically-released",
```

Comment:
💥  I'm unsure if installs really work this way. I mean they are not really in that version, but in another one, aren't they? Why can't we have the right numbers here and have them updated by lerna?